content/en/continuous_delivery/deployments/argocd.md

```diff
@@ -144,6 +144,35 @@ metadata:
     dd_customtags: "region:us1-east, team:backend"
 ```
 
+## Correlate deployments with CI pipelines
+
+If Argo CD deployments are triggered from a CI pipeline, you may correlate the deployment execution and the pipeline,
+allowing to check the associated deployments to a pipeline and the pipeline that triggered a deployment from Datadog.
+
+In this moment, not all the setups are supported. To be able to make the correlation your setup has to be similar to this:
```

Comment:
```suggestion
To correlate your deployments with your pipeline, your CI pipeline must make its changes to the Git repository with `git commit` as shown below:
```

Reply:
As a general rule, the documentation emphasizes what **is** supported, rather than what is **not** supported.

Since not all users are able to view images, I added a suggestion here to describe the key requirement for this setup to work. Feel free to make changes if my suggestion is not accurate.